uportal-war/src/main/webapp/media/skins/respondr/defaultSkin.less

```diff
@@ -30,10 +30,9 @@
  * See https://wiki.jasig.org/display/UPC/Respondr+Skinning+Process for instructions on creating a custom skin.
  */
 
-/** DO NOT REMOVE THIS INCLUDE ****************/
-@import "defaultSkin/less/variables.less"; 
-/**********************************************/
-
+/***** DO NOT REMOVE THESE INCLUDES *****/
+@import "defaultSkin/less/variables.less";
 @import "common/common.less";
+/****************************************/
```

Comment:
Beneath this line, maybe we should add...

/**
 * To  create your own skin based on defaultSkin, uncomment the following line
 * and replace {skinName} with the name of your skin.
 */
/* @import "{skinName}/less/variables.less"; */